backend/src/app.module.ts

```diff
@@ -1,10 +1,41 @@
 import { Module } from '@nestjs/common';
-import { AppController } from './app.controller';
-import { AppService } from './app.service';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { CommandsModule } from './commands.module/commands.module';
+import { Item } from './models/item.model';
+import { QueriesModule } from './queries.module/queries.module';
 
 @Module({
-  imports: [],
-  controllers: [AppController],
-  providers: [AppService],
+  imports: [
+    ConfigModule.forRoot({
+      envFilePath: '../.env',
+      isGlobal: true,
+    }),
+    TypeOrmModule.forRootAsync({
+      imports: [ConfigModule],
+      useFactory: (configService: ConfigService) => {
+        return {
+          type: 'mongodb',
+          authSource: 'admin',
+          host: configService.get('DB_MONGO_HOST'),
+          port: +configService.get('DB_MONGO_PORT'),
+          username: configService.get('DB_MONGO_USER'),
+          password: configService.get('DB_MONGO_PW'),
+          database: configService.get('DB_MONGO_NAME'),
+          // TODO: Remove this option in production:
+          // Only enable this option if your application is in development,
+          // otherwise use TypeORM migrations to sync entity schemas:
+          // https://typeorm.io/#/migrations
+          synchronize: true,
```

Comment:
Add a check for this using the NODE_ENV environment variable

If the variable is set to 'production' then set synchronize to false